api/src/main/java/org/apache/iceberg/TableScan.java

```diff
@@ -148,7 +89,9 @@ default TableScan select(String... columns) {
    * @return a table scan which can read append data from {@code fromSnapshotId}
    * exclusive and up to {@code toSnapshotId} inclusive
    */
-  TableScan appendsBetween(long fromSnapshotId, long toSnapshotId);
```

Comment:
I think we should probably deprecate this because we want people to move to incremental. (Eventually)

Reply:
yes, once new IncrementAppendScan is implemented, we can mark these two appends methods as deprecated